README.md

```diff
@@ -1,57 +1,63 @@
 # Guide
 
-A website to teach Wagtail CMS to content editors, moderators and administrators.
+Guide is a website to help content editors, moderators, administrators, and other users learn how to user the Wagtail content management system (CMS).
 
-Wagtail guide brings you:
-- Tutorials, 
-- How-to, 
-- Reference 
-- Background information
+The Wagtail guide will ultimately include:
+- Tutorials 
+- How-to articles
+- Reference materials
+- Walkthroughs and visual learning materials
```

Comment:
```suggestion
The Wagtail guide will ultimately include:
- Tutorials
- How-to
- Explanation
- Reference

Learn more about the [documentation system](https://documentation.divio.com/).
```

Reply:
So I added nouns "articles"/"materials" to "How-To" and "Reference" because "How-to" is a compound adjective that needs a noun to make the grammar work and the singular version of "Reference" would be incorrect for the meaning we're aiming for. The plural "References" also doesn't work because when most English-speaking people think of "references", they think of citations or job "references". If you look in Divio's copy, they do a similar thing. The singular "Explanation" also doesn't work because it implies the guide will only contain a single explanation. So I provided examples of what those explanations would be instead to give people a better idea what it would contain.

I'm going to keep the phrasing, but I think the addition of the link is a good idea.